libs/go/libp2p_node/dht/dhtpeer/dhtpeer_test.go

```diff
@@ -1814,10 +1815,15 @@ func SetupDelegateClient(
 	client.PoR = signature
 
 	client.Rx = make(chan *aea.Envelope, 2)
-	client.Conn, err = net.Dial("tcp", host+":"+strconv.FormatInt(int64(port), 10))
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+	client.Conn, err = tls.Dial("tcp", host+":"+strconv.FormatInt(int64(port), 10), conf)
 	if err != nil {
 		return nil, nil, err
 	}
+	// Read signature, skip it here
```

Comment:
Why skip?

Reply:
proper check added

Reply:
So the comment is no longer correct...please update

Reply:
done